src/helpers/common.ts

```diff
@@ -16,11 +16,14 @@ export const generatePrivateKey = (ecCurve: EC, buf: typeof Buffer): Buffer => {
   return ecCurve.genKeyPair().getPrivate().toArrayLike(buf);
 };
 
+const secp256k1EC = new EC("secp256k1");
+const ed25519EC = new EC("ed25519");
```

Comment:
this was intentionally not left as a global variable to reduce the build size

Reply:
would create EC object every time increase memory usage and latency?